src/main/kotlin/com/atlassian/performance/tools/jiraactions/webdriver/WebDriverUtils.kt

```diff
@@ -0,0 +1,48 @@
+package com.atlassian.performance.tools.jiraactions.webdriver
+
+import org.openqa.selenium.JavascriptExecutor
+import org.openqa.selenium.NoSuchElementException
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.ExpectedCondition
+import java.util.concurrent.atomic.AtomicInteger
+
+class WebDriverUtils {
+    companion object {
+        fun <T> executeScript(driver: WebDriver, js: String): T {
+            driver as JavascriptExecutor
+            return driver.executeScript(js) as T
+        }
+
+        fun sometimesApply(condition: ExpectedCondition<Boolean>): ExpectedCondition<Boolean> {
```

Comment:
I left the sometimes* methods here although they are technically used in a different PR. I struggle a bit with my workspace, there are too many interrelated changes there.

Reply:
Which PR is that?

Reply:
Not published yet, I want to get rid of all the other changes in my workspace first.

Reply:
Note that this is not API and therefore won't be consumable outside of `jira-actions`. Unused internal code begs to be deleted. It's best to combine both new code and it's usages in the same commit. It's easier to understand in the future.

Reply:
Can I make it an API? I'd like to use it in software-actions too.

Reply:
Sure, gotta move it into an `api` subpackage